.evergreen/generate_evergreen_tasks.js

```diff
@@ -410,21 +396,21 @@ AWS_AUTH_VERSIONS.forEach(VERSION => {
       { func: 'run aws ECS auth test' }
     ]
   });
-})
-
+});
 
 const BUILD_VARIANTS = [];
 
 const getTaskList = (() => {
   const memo = {};
-  return function(mongoVersion) {
-    const key = mongoVersion;
+  return function (mongoVersion, onlyFullSuite = false) {
```

Comment:
I thought we removed the `full-suite` tag in the 3.6 PR?

Reply:
Oops I missed porting that over, thanks!